server.js

```diff
@@ -9,10 +11,47 @@ app.use(cors());
 
 const PORT = process.env.PORT || 3001;
 
-app.get('/test', (request, response) => {
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017/can-books-database');
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function () {
+  console.log('Mongoose is connected');
+});
+
+const BookModel = mongoose.model('book-collection', bookSchema);
```

Comment:
Nice work so far on this! I would double check first arg to mongoose.model(). This hyphen may cause some issues. I think JB mentioned this in class but this name should be singular and typically it is uppercase I believe. 

See docs here: [Constructing Documents](https://mongoosejs.com/docs/models.html)

 I am not sure if you are using this one anyway as you have a different arg to mongoose.model() in `BookSchema.js`